test/inductor/test_torchinductor_codegen_dynamic_shapes.py

```diff
@@ -255,7 +255,7 @@ def run(*ex, **kwargs):
     "test_split_cumprod_low_prec_dynamic_shapes": TestFailure(("cpu",)),
     "test_split_dynamic_shapes": TestFailure(("cpu", "cuda", "xpu")),
     "test_topk_dynamic_shapes": TestFailure(("cpu", "cuda", "xpu")),
-    "test_unbind_dynamic_shapes": TestFailure(("cpu", "cuda", "xpu")),
+    "test_unbind_dynamic_shapes": TestFailure(("cpu", "xpu")),
```

Comment:
For this case, here is the change:
```python
    return (reinterpret_tensor(arg0_1, (4, 4), (4, 1), 0), reinterpret_tensor(arg0_1, (4, 4), (4, 1), 16), reinterpret_tensor(arg0_1, (4, 4), (4, 1), 32), reinterpret_tensor(arg0_1, (4, 4), (4, 1), 48), reinterpret_tensor(arg0_1, (4, 4), (16, 4), 0), reinterpret_tensor(arg0_1, (4, 4), (16, 4), 1), reinterpret_tensor(arg0_1, (4, 4), (16, 4), 2), reinterpret_tensor(arg0_1, (4, 4), (16, 4), 3), )
```
-> 
```python
    buf0 = empty_strided_cpu((4, 4), (16, 4), torch.float32)
    buf1 = empty_strided_cpu((4, 4), (16, 4), torch.float32)
    buf2 = empty_strided_cpu((4, 4), (16, 4), torch.float32)
    buf3 = empty_strided_cpu((4, 4), (16, 4), torch.float32)
    cpp_fused_unbind_0(arg0_1, buf0, buf1, buf2, buf3)
    return (reinterpret_tensor(arg0_1, (4, 4), (4, 1), 0), reinterpret_tensor(arg0_1, (4, 4), (4, 1), 16), reinterpret_tensor(arg0_1, (4, 4), (4, 1), 32), reinterpret_tensor(arg0_1, (4, 4), (4, 1), 48), buf0, buf1, buf2, buf3, )
```

Reply:
right, so there is another kernel. we need to look into why that occurs.